client/src/botany_client/client.py

```diff
@@ -147,6 +148,16 @@ def wrapped(board, move_list, token, state):
     if opcode_limit == 0:
         opcode_limit = None
 
+    #Initiate the move list parameter
+    if move_list is None:
+        move_list = []
+    else:
+        assert move_list.isdigit(), "move-list should only consist of integers"
+        move_list = list(move_list)
+        move_list = [int(move) for move in move_list]
+        assert all(a < game.NCOLS for a in move_list ), f"Move cannot be greater then {game.NCOLS-1}"
```

Comment:
This `assert` is also wrong, as it is again specific to `botany_connectfour.game`.

Reply:
Was thinking at ConnectFour only.